docs/appkit/unity/core/installation.mdx

```diff
@@ -108,3 +109,10 @@ public void OpenAppKitModal()
     AppKit.OpenModal();
 }
 ```
+
+## Examples and Test dApps
+
+- [Unity AppKit Example in Github](https://github.com/reown-com/reown-dotnet/tree/main/sample/)
+- [A sample dApp demonstrating AppKit integration in Unity](https://unity-appkit.vercel.app/)
```

Comment:
Don't know if it can be called a `dApp` as it doesn't have any smart contracts 😄 Let's focus on the platform
```suggestion
- [A WebGL sample](https://unity-appkit.vercel.app/)
```